core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/worker/DataProcessor.scala

```diff
@@ -329,8 +330,47 @@ class DataProcessor(
           )
           outputIterator.appendSpecialTupleToEnd(FinalizeOperator())
         }
-      case marker: EpochMarker =>
-        epochManager.processEpochMarker(channel.from, marker, this)
+    }
+  }
+
+  def processEpochMarker(
+      channelId: ChannelID,
+      marker: ChannelMarkerPayload,
+      logManager: ReplayLogManager
+  ): Unit = {
+    val markerId = marker.id
+    val command = marker.commandMapping.getOrElse(actorId, null)
```

Comment:
use get() which returns a None when not found? I want to remove `null` if possible.

Reply:
done